### PR TITLE
Night Mode and More Hotkeys

### LINKED ISSUE #6212: Popular request: create a Night Mode with dark CSS
Here's an example of a "night mode wikipedia app": 

Some campers have created unofficial night mode skins for FCC, and I think we can create a good one that can be toggled on campers' code portfolios.

### LINKED ISSUE #6385: Ability to toggle night mode on and off from options

### PR DESCRIPTION
**Pull request by benmcmahon100**
Implementation of night mode and the new hot keys. Massive credit to @hallaathrad , @raisedadead , and eveyone else who helped in the design and creation process. Unfortunately the git history on the initial branch was all over the place so it cannot be used.

Closes #6212
Closes #6385

### CODE REVIEW

File: client/less/challenge.less [hunk 1]
```diff
@@ -111,3 +111,67 @@ iframe.iphone {
...
+      color:#5CAFD6;
+    }
+    .cm-property, .cm-string {
+      color:#B5753A;
+    }
+    .cm-keyword, .cm-attribute {
+      color:#9BBBDC;
+    }
+  }
+}
```
Comment (hallaathrad) [inline client/less/challenge.less]: why is all this stuff repeated 3 times?

File: client/less/challenge.less [hunk 2]
```diff
@@ -111,3 +111,67 @@ iframe.iphone {
     right: -207px;
   }
 }
+
+
+.night {
```
Comment (hallaathrad) [inline client/less/challenge.less]: this should only be once, in the main.less

Comment (benmcmahon100) [inline client/less/challenge.less]: That is what was in the diff on the original pr. I can adjust it so that it is just one though as well

File: client/main.js [hunk 3]
```diff
@@ -356,7 +356,7 @@ $(document).ready(function() {
 
   function showMap() {
     if (!main.isMapAsideLoad) {
-      var mapAside = $('<iframe>');
+      var mapAside = $('<iframe id = "map-aside-frame" >');
```
Comment (hallaathrad) [inline client/main.js]: remove the unnecessary whitespace `<iframe id="blah">`
